forge-gui/res/cardsfolder/upcoming/champions_of_tyr.txt

```diff
@@ -0,0 +1,14 @@
+Name:Champions of Tyr
+ManaCost:2 W W
+Types:Creature Angel Knight
+PT:4/3
+K:Flying
+K:Double Team
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ DBDelayedTrigger | TriggerDescription$ When CARDNAME enters the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with your choice of a +1/+1 counter, a flying counter, or a lifelink counter on it."
```

Comment:
I do have a PR in progress for boons if you want to set this one aside until we get that nailed down. Completely up to you, I can also tweak it once it's in.